confection/tests/test_config.py

```diff
@@ -424,6 +424,31 @@ def catsie_567(*args: Optional[str], foo: str = "bar"):
     assert my_registry.resolve(cfg)["config"] == "^_^"
 
 
+def test_fill_config_positional_args_w_promise():
+    @my_registry.cats("catsie.v568")
+    def catsie_568(*args: str, foo: str = "bar"):
+        assert args[0] == "^(*.*)^"
+        assert foo == "baz"
+        return args[0]
+
+    @my_registry.cats("cat_promise.v568")
+    def cat_promise() -> str:
+        return "^(*.*)^"
+
+    cfg = {
+        "test_fn": {"@cats": "catsie.v568", "*": {"test_arg": {"@registry": "factory"}}}
+    }
```

Comment:
This doesn't seem to be used? Was an additional test intended here?

Reply:
You are right. I must have copied it over to make sure that the structure was correct in the test. I have deleted it now.